sdks/python/apache_beam/testing/load_tests/co_group_by_key_test.py

```diff
@@ -18,7 +18,9 @@
 To run test on DirectRunner
 
 python setup.py nosetests \
-    --test-pipeline-options="--input_options='{
+    --test-pipeline-options="
+      --metrics_project_id=big-query-project
```

Comment:
Well, I wasn't sure about it because `project_id` exists in case of choosing `Dataflow Runner`. I thought there can be a case when someone has a different BigQuery project for publishing metrics. WDYT?

Reply:
I think it's better to have one option for this, if possible. I don't think using multiple google cloud project will be ever needed. Even if so, then adding a new, big query specific pipeline option will be easy.